.circleci/config.yml

```diff
@@ -42,13 +42,26 @@ jobs:
       - checkout
       - run: git submodule sync
       - run: git submodule update --init
+      # Recipe: https://pre-commit.com/#managing-ci-caches
+      - run:
+          name: Combine pre-commit config and python versions for caching
+          command: |
+            cp common/pre-commit-config.yaml pre-commit-cache-key.txt
+            python --version --version >> pre-commit-cache-key.txt
+      - restore_cache:
+          keys:
+          - v1-pc-cache-{{ checksum "pre-commit-cache-key.txt" }}
```

Comment:
Looks like this prefix can be anything, we should probably use something more descriptive and related to pre-commit here.

Reply:
ah yeah, this is a copy-paste from the original example... but agreed... pc seems to stand for both "personal computer", "political correctness" and now also "pre-commit" :smile: